draft-irtf-nwcrg-coding-and-congestion-04.txt

```diff
@@ -327,25 +369,32 @@ Internet-Draft            Coding and congestion                July 2020
 3.2.  Discussion
 
    While CC is in principle independent of other transport functions
-   such as reliability, we note that CC is often embedded in reliable
-   transfer protocols (e.g.  TCP).  This approach may require that the
-   transport protocol does not implement a fully reliable data transfer
+   such as connection establishment or reliability, we note that CC can
+   be coupled with a reliable mechanism.  This approach may require that
+   the transport protocol does not implement a fully reliable data
+   transfer service (e.g., based on lost packet retransmission),
+   otherwise this could lead to spurious retransmissions.  UDP is an
```

Comment:
I'd remove the "otherwise this could lead to spurious retransmissions" part: it confuses me. Do you have a concrete case in mind? If so, you may want to elaborate and cite the right reference.... but otherwise....   if a retransmission is unnecessary from the FEC point of view, that's not what so many other IETF documents call a spurious loss event (which would be coupled with spuriously retransmitting packets). Anyway, adding FEC on a completely reliable channel is useless, so that's enough to say.

Reply:
You are right - this has been reworked in the commit in #29